contracts/src/state/operatorsRegistry/Operators.2.sol

```diff
@@ -143,20 +145,8 @@ library OperatorsV2 {
                 ++idx;
             }
         }
-
-        Operator[] memory activeOperators = new Operator[](activeCount);
-
-        uint256 activeIdx = 0;
-        for (uint256 idx = 0; idx < operatorCount;) {
-            if (r.value[idx].active) {
-                activeOperators[activeIdx] = r.value[idx];
-                unchecked {
-                    ++activeIdx;
-                }
-            }
-            unchecked {
-                ++idx;
-            }
+        assembly ("memory-safe") {
+            mstore(activeOperators, activeCount)
         }
```

Comment:
Do we have sufficient tests to ensure that this assembly code is robust ?

Looking at the test file we seem to have 2 that cover listActiveOperators()?

Reply:
@iamsahu do you feel like we have enough test coverage here ?

Reply:
great, thanks for adding in more tests here